controllers/nodeagent.go

```diff
@@ -25,14 +26,24 @@ import (
 )
 
 const (
-	ResticRestoreHelperCM = "restic-restore-action-config"
-	FsRestoreHelperCM     = "fs-restore-action-config"
-	HostPods              = "host-pods"
-	HostPlugins           = "host-plugins"
+	ResticRestoreHelperCM   = "restic-restore-action-config"
+	FsRestoreHelperCM       = "fs-restore-action-config"
+	HostPods                = "host-pods"
+	HostPlugins             = "host-plugins"
+	Cluster                 = "cluster"
+	IBMCloudPlatform        = "IBMCloud"
+	GenericPVHostPath       = "/var/lib/kubelet/pods"
+	IBMCloudPVHostPath      = "/var/data/kubelet/pods"
```

Comment:
I think this should be deleted

what IBM changes is just host-plugins path, no?

Reply:
I think yesterday in scrum we discussed that we need both.
cc: @sseago

Reply:
I recall both are needed to allow a symlink to work.  Pod and Host iiuc

Reply:
what I remember was that we need to mount both `/var/lib/kubelet` and `/var/data/kubelet` on pod

I tested only changing host-plugins path and e2e block datamover worked for me in IBM (which never worked before)

If we need both, then velero docs are still wrong https://velero.io/docs/v1.14/csi-snapshot-data-movement/#configure-node-agent-daemonset-spec

Reply:
yeah we might need to update the docs upstream